src/Sarif.Converters/ToolFormat.cs

```diff
@@ -14,8 +14,10 @@ public enum ToolFormat
         ClangAnalyzer,
         /// <summary>CppCheck's file format.</summary>
         CppCheck,
-        /// <summary>Fortify's file format.</summary>
+        /// <summary>Fortify's report file format.</summary>
         Fortify,
+        /// <summary>Fortify's FPR file format.</summary>
+        FortifyFpr,
```

Comment:
>FortifyFpr [](start = 8, length = 10)

I wonder if the FPR shouldn't be given the core 'Fortify' value and the Fortify report could be granted 'FortifyReport'. The FPR is the really comprehensive thing and, generally, what we would prefer to convert.